tests/integration/IntegrationTestCase.php

```diff
@@ -32,21 +32,21 @@ protected function createMatejInstance(): Matej
         return $instance;
     }
 
-    protected function assertResponseCommandStatuses(Response $response, ...$expectedCommandStatuses): void
+    protected static function assertResponseCommandStatuses(Response $response, ...$expectedCommandStatuses): void
```

Comment:
Why is this static? This should be non-static, co it could be used in a same way as other phpunit asserts (via $this->assertResponseCommandStatuses, not static::assertResponseCommandStatuses)

Reply:
This is what happens then:
![static](https://user-images.githubusercontent.com/793041/38867133-6f00c772-4243-11e8-9f94-b4ca099ecc2f.png)

So if we don't need to, lets keep this non-static :)